scripts/localnet.sh

```diff
@@ -8,7 +8,7 @@ BASE_DIR="$SCRIPT_DIR/.."
 
 : "${CHAIN:=local}"
 : "${BUILD_BINARY:=1}"
-: "${FEATURES:=pow-faucet}"
+: "${FEATURES:="pow-faucet runtime-benchmarks"}"
```

Comment:
Do we need the `runtime-benchmarks` for localnet ?

Reply:
Yeah it didn't work, got the same error that you DM'd me about